odonto/odonto_submissions/models.py

```diff
@@ -122,7 +122,7 @@ def send(self):
             )
         current_submission.state = Submission.SENT
         current_submission.save()
-        send_message.send_message(current_submission.xml)
+        dpb_api.send_message(current_submission.xml)
```

Comment:
What actually calls this method ?

Reply:
That's coming in a future PR I'm afraid 😞